chart/templates/deployment-app.yaml

```diff
@@ -57,6 +64,11 @@ spec:
             initialDelaySeconds: 15
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+          volumeMounts:
+            - mountPath: /etc/ots/customize.yaml
+              subPath: customize.yml
```

Comment:
You could just mount the whole secret to `/etc/ots` in order not to have to modify this when more files are to be mounted in. [opt]